src/components/MDXRenderer.tsx

```diff
@@ -8,6 +8,8 @@ import rehypeMathJax from 'rehype-mathjax/svg'
 import remarkGfm from 'remark-gfm'
 import MCQBlock from './MCQBlock'
 import FITBBlock from './FITBBlock'
+import FITB_Text from './FITB_Text'
+import FITB_Blank from './FITB_Blank'
```

Comment:
I'm not a fan of using underscores in component names since the convention is to use Pascal case.